keyboards/kinesis/keymaps/TuesdayJohn/config.h

```diff
@@ -0,0 +1,13 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+
+#include "../../config.h"
```

Comment:
Line 5 can be safely deleted. QMK's build script automatically includes all the `config.h` files in the build tree, so this line is redundant.

```suggestion
```